packages/@aws-cdk/aws-globalaccelerator/README.md

```diff
@@ -5,133 +5,186 @@
+Balancers, EC2 Instances and Elastic IP Addresses.
 
-## Accelerator
+For more information, see the [AWS Global
+Accelerator Developer Guide](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_GlobalAccelerator.html).
 
-The `Accelerator` resource is a Global Accelerator resource type that contains information about how you create an accelerator. An accelerator includes one or more listeners that process inbound connections and direct traffic to one or more endpoint groups, each of which includes endpoints, such as Application Load Balancers, Network Load Balancers, and Amazon EC2 instances.
+## Example
 
-To create the `Accelerator`:
+Here's an example that sets up a Global Accelerator for two Application Load
+Balancers in two different AWS Regions:
 
 ```ts
 import globalaccelerator = require('@aws-cdk/aws-globalaccelerator');
+import ga_endpoints = require('@aws-cdk/aws-globalaccelerator-endpoints');
+import elbv2 = require('@aws-cdk/aws-elasticloadbalancingv2');
 
-new globalaccelerator.Accelerator(stack, 'Accelerator');
+// Create an Accelerator
+const accelerator = new globalaccelerator.Accelerator(stack, 'Accelerator');
+
+// Create a Listener
+const listener = accelerator.addListener('Listener', {
+  portRanges: [
+    { fromPort: 80 },
+    { fromPort: 443 },
+  ],
+});
+
+// Import the Load Balancers
+const nlb1 = elbv2.NetworkLoadBalancer.fromNetworkLoadBalancerAttributes(stack, 'NLB1', {
+  loadBalancerArn: 'arn:aws:elasticloadbalancing:us-west-2:111111111111:loadbalancer/app/my-load-balancer1/e16bef66805b',
+});
+const nlb2 = elbv2.NetworkLoadBalancer.fromNetworkLoadBalancerAttributes(stack, 'NLB2', {
+  loadBalancerArn: 'arn:aws:elasticloadbalancing:ap-south-1:111111111111:loadbalancer/app/my-load-balancer2/5513dc2ea8a1',
+});
 
+// Add one EndpointGroup for each Region we are targeting
+listener.addEndpointGroup('Group1', {
+  endpoints: [new ga_endpoints.NetworkLoadBalancerEndpoint(nlb1)],
+});
+listener.addEndpointGroup('Group2', {
+  // Imported load balancers automatically calculate their Region from the ARN.
+  // If you are load balancing to other resources, you must also pass a `region`
+  // parameter here.
+  endpoints: [new ga_endpoints.NetworkLoadBalancerEndpoint(nlb2)],
+});
 ```
 
-## Listener
+## Concepts
+
+The **Accelerator** construct defines a Global Accelerator resource.
+
+An Accelerator includes one or more **Listeners** that accepts inbound
+connections on one or more ports.
+
+Each Listener has one or more **Endpoint Groups**, representing multiple
+geographically distributed copies of your application. There is one Endpoint
+Group per Region, and user traffic is routed to the closest Region by default.
+
+An Endpoint Group consists of one or more **Endpoints**, which is where the
+user traffic coming in on the Listener is ultimately sent. The Endpoint port
+used is the same as the traffic came in on at the Listener, unless overridden.
+
+## Types of Endpoints
```

Comment:
Curious: Why you've chosen this way, instead of a brief summary here and then documenting the actual list in the endpoints package?

Reply:
Yeah, it wasn't an easy choice. Ultimately I decided that looking in 1 place is better for users than forcing them to look in 2 places, and the list wasn't impressive enough that we'd gain clarity from splitting it up.

Reply:
Sounds good.

I was curious because I've done the opposite with APIGWv2.